x-pack/plugin/security/qa/smoke-test-all-realms/src/javaRestTest/resources/jwt/README.asciidoc

```diff
@@ -0,0 +1,57 @@
+[2022-02-24T01:17:47,778][INFO ][o.e.x.s.a.j.JwtRealmTests] [testCreateJwtSmokeTestRealm] {Authorization=Bearer eyJhbGciOiJIUzI1NiJ9.eyJhdWQiOiJhdWQ4Iiwic3ViIjoic2VjdXJpdHlfdGVzdF91c2VyIiwicm9sZXMiOiJbc2VjdXJpdHlfdGVzdF9yb2xlXSIsImlzcyI6ImlzczgiLCJleHAiOjQwNzA5MDg4MDAsImlhdCI6OTQ2Njg0ODAwfQ.YbMbSEY8j3BdE_M71np-5Q9DFHGcjZcu7D4Kk1Ji0wE, X-Client-Authentication=SharedSecret client-shared-secret-string}
+[2022-02-24T01:17:47,789][INFO ][o.e.x.s.a.j.JwtRealmTests] [testCreateJwtSmokeTestRealm] TOKEN[iss8/aud8/security_test_user]: jwt=[eyJhbGciOiJIUzI1NiJ9.eyJhdWQiOiJhdWQ4Iiwic3ViIjoic2VjdXJpdHlfdGVzdF91c2VyIiwicm9sZXMiOiJbc2VjdXJpdHlfdGVzdF9yb2xlXSIsImlzcyI6ImlzczgiLCJleHAiOjQwNzA5MDg4MDAsImlhdCI6OTQ2Njg0ODAwfQ.YbMbSEY8j3BdE_M71np-5Q9DFHGcjZcu7D4Kk1Ji0wE], secret=[client-shared-secret-string].
+[2022-02-24T01:17:47,883][INFO ][o.e.x.s.a.j.JwtRealmTests] [testCreateJwtSmokeTestRealm] STATS: expected=[jwt8]
+   realm[jwt8,8/1], stats=[{name=jwt8, order=8}]
+[2022-02-24T01:17:47,884][INFO ][o.e.x.s.a.j.JwtRealmTests] [testCreateJwtSmokeTestRealm] RESULT: expected=[jwt8]
+  realms=[1], expected=[jwt8,8], current[jwt8,8], status=[SUCCESS], authenticated=[true], msg=[null], meta=[{}], user=[User[username=security_test_user,roles=[security_test_role],fullName=null,email=null,metadata={jwt_claim_iss=iss8, jwt_claim_sub=security_test_user, jwt_claim_aud=[aud8], jwt_claim_roles=[security_test_role]}]].
+[2022-02-24T01:17:47,884][INFO ][o.e.x.s.a.j.JwtRealmTests] [testCreateJwtSmokeTestRealm] Test succeeded
+[2022-02-24T01:17:47,887][INFO ][o.e.x.s.a.j.JwtRealmTests] [testCreateJwtSmokeTestRealm]
+===
+Request Headers
+===
+X-Client-Authentication: client-shared-secret-string
+Authorization: eyJhbGciOiJIUzI1NiJ9.eyJhdWQiOiJhdWQ4Iiwic3ViIjoic2VjdXJpdHlfdGVzdF91c2VyIiwicm9sZXMiOiJbc2VjdXJpdHlfdGVzdF9yb2xlXSIsImlzcyI6ImlzczgiLCJleHAiOjQwNzA5MDg4MDAsImlhdCI6OTQ2Njg0ODAwfQ.YbMbSEY8j3BdE_M71np-5Q9DFHGcjZcu7D4Kk1Ji0wE
+
+===
+elasticsearch.yml settings
+===
+xpack.security.authc.realms.jwt.jwt8.order: 8
+xpack.security.authc.realms.jwt.jwt8.allowed_issuer: iss8
+xpack.security.authc.realms.jwt.jwt8.allowed_signature_algorithms: [HS256]
+xpack.security.authc.realms.jwt.jwt8.allowed_audiences: [aud8]
+xpack.security.authc.realms.jwt.jwt8.claims.principal: sub
+xpack.security.authc.realms.jwt.jwt8.client_authentication.type: SharedSecret
+
+===
+PKC JWKSet contents
+===
+Not found.
+
+===
+elasticsearch-keystore secure settings
+===
+xpack.security.authc.realms.jwt.jwt8.hmac_key: hmac-oidc-key-string-for-hs256-algorithm
+xpack.security.authc.realms.jwt.jwt8.client_authentication.shared_secret: client-shared-secret-string
+
```

Comment:
I think we only need this section in this file. Rest is noise and it's easier to understand the point without it.

Reply:
Resolved.